torch/csrc/Generator.cpp

```diff
@@ -116,37 +116,60 @@ static PyObject* THPGenerator_setState(PyObject* _self, PyObject* _new_state) {
   END_HANDLE_TH_ERRORS
 }
 
-static PyObject* THPGenerator_manualSeed(PyObject* _self, PyObject* seed) {
-  HANDLE_TH_ERRORS
-  auto self = (THPGenerator*)_self;
-  auto generator = self->cdata;
-  THPUtils_assert(
-      THPUtils_checkLong(seed),
-      "manual_seed expected a long, "
-      "but got %s",
-      THPUtils_typename(seed));
-  // See Note [Acquire lock when using random generators]
-  std::lock_guard<std::mutex> lock(generator.mutex());
+uint64_t unpack_int_pyobj(PyObject* obj) {
   // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
-  uint64_t seed_unpacked;
+  uint64_t obj_unpacked;
```

Comment:
nitpick: The "unsigned" being default named as obj_unpacked, and the signed being named "obj_unpacked_signed" is kinda backwards from the c++ convention of unsigned being named and signed not being named.

Nitpick #2: Try-catch for overflow like this can be improved to use PyError

```
uint64_t obj_unpacked;
if (PyLong_Check(obj)) {
  uint64_t value = PyLong_AsUnsignedLongLong(obj);
  if (value == (uint64_t)-1 && PyErr_Occurred()) {
    PyErr_Clear();
    obj_unpacked = *(reinterpret_cast<uint64_t*>(&THPUtils_unpackLong(obj);));
  } else {
    obj_unpacked = value;
  }
} else {
  obj_unpacked = THPUtils_unpackUInt64(obj);
}
```